recipe/bld.bat

```diff
@@ -4,19 +4,24 @@ if errorlevel 1 exit 1
 cd build
 if errorlevel 1 exit 1
 
+set BUILDCONF=Release
+
 cmake -G Ninja ^
-    -D CMAKE_BUILD_TYPE=Release ^
-    -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -D CMAKE_BUILD_TYPE=%BUILDCONF% ^
+    -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%\qgis ^
```

Comment:
I think this is the reason why the you have to fiddle with the `%PATH%`. Shouldn't this be:
```
CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
```
?